src/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"baseUrl": ".",
+		"outDir": "../dist",
+		"composite": true,
+		"tsBuildInfoFile": "../dist/tsconfig.tsbuildinfo",
+		"paths": {
+			"#lib/*": ["lib/*"]
```

Comment:
isn't this missing a generated path for importing the JSON file? https://github.com/skyra-project/teryl/blob/main/src/tsconfig.json#L11

On that note, we should import the JSON as a JSON module.
```ts
import xkcdData from '#generated/data/xkcd.json' assert { type: 'json' };
```

Reply:
Import assertions regressed to Stage 2, the stability is not longer guaranteed.

Plus, imports cache the raw JSON string, while FS + JSON.parse does not.